frontend/src/stories/Introduction.stories.mdx

```diff
@@ -4,12 +4,24 @@ import { Meta } from '@storybook/addon-docs';
 
 # Design System
 
-At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation in Angular or Rails.
+At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation.
 
 Our goal is to reach a point where every view in OpenProject is built with elements described in the design system. However, we are aware that reaching this goal will take time for a tool as complex and layered as OpenProject.
 
 As such, the design system is still in its infancy and will grow with each release.
 
+## Purpose (what it is and what it's not)
+
+This Storybook-based design system documentation only seeks to describe the actual reusable components; it does not seek to document features where these components are used.
+
+The purpose of this document is to:
+
+- Have a single point of truth on what reusable components should look like, how they should behave, and what options they provide.
```

Comment:
@psatyal I struggle again with the absoluteness of "single point of truth". I replaced it with "central documentation". We want that everyone knows where to look for guidance and orientation. The DS can never have the single point of truth, simply because it is and will always be incomplete and insufficient for describing many singular cases. To put it differently, I don't think that I'd agree that every little deviant to what is available in the DS needs to go through a DS process. I fear that we stop innovating and just be busy with ourselves.

```suggestion
- Have a central documentation of reusable components specifying how they should look like, how they should behave, how they shall get used and what options they provide.
```

Reply:
This is mostly a naming thing, I think. "Central documentation" basically means single point of truth, no?

I also think it should not be a documentation of what they *should* look like, but what they *do* look like, but I guess this ties in to our question on whether we want to write down design specifics instead of specifying those in tickets and letting the designs speak for themselves in stories.

Reply:
Again a point here if we use only "components" or "components and patterns" in the explanations.

About the central documentation I thing it makes it less strong using this term.

Reply:
@b12f Yes, I agree with changing it to "_do_ look like". It means that whatever is defined here in the DS is implemented and meant to be used right now. 

This also means that we do not specify here how something should look like in the future. @psatyal  I think we should put that differentiation somewhere here in the "Purpose".

@marcalcobe "Single point of truth" is much more absolute than a _documentation_ or a _catalogue_ or simply a _system_. It means that it contains all(!) the truth as it is even the single point of truth. And that is impossible to achieve. OpenProject has so much more details and decisions that are not part and will never be part of the design system. For example: If there is a feature that is a bit different to what is implemented in the DS because that deviation makes the feature better but at the same time we do not think that this will be a reusable pattern, the DS will not contain that deviation. Still, the truth is that the feature with the deviation exists. Consequently the name "single point of truth" does not work. I personally would call it a _design system_. And I would put what it means to us in this section here and not in the name.

Reply:
Removed references to "single point of truth" and and replaced "should" with how things actually are. I also made this clearer in the the _this document does not_ section. 

Also use "components and patterns" instead of just "components".